docs/integrations/hdinsight.md

```diff
@@ -0,0 +1,182 @@
+
+mkdir -p /usr/lib/hopsworks
+chown root:hadoop /usr/lib/hopsworks
+cd /usr/lib/hopsworks
+
+curl -o client.tar.gz -H "Authorization: ApiKey ${API_KEY}" https://$HOST/hopsworks-api/api/project/$PROJECT_ID/client
+
+tar -xvf client.tar.gz
+tar -xzf client/apache-hive-*-bin.tar.gz
+mv apache-hive-*-bin apache-hive-bin
+rm client.tar.gz
+rm client/apache-hive-*-bin.tar.gz
+
+curl -H "Authorization: ApiKey ${API_KEY}" https://$HOST/hopsworks-api/api/project/$PROJECT_ID/credentials | jq -r .kStore | base64 -d > keyStore.jks
+
+curl -H "Authorization: ApiKey ${API_KEY}" https://$HOST/hopsworks-api/api/project/$PROJECT_ID/credentials | jq -r .tStore | base64 -d > trustStore.jks
+
+echo -n $(curl -H "Authorization: ApiKey ${API_KEY}" https://$HOST/hopsworks-api/api/project/$PROJECT_ID/credentials | jq -r .password) > material_passwd
+
+chown -R root:hadoop /usr/lib/hopsworks
+```
+
+## Step 3: Configure HDInsight for Feature Store access
+
+The Hadoop and Spark installations of the HDInsight cluster need to be configured in order to access the Feature Store. This can be achieved either by using a [bootstrap script](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-bootstrap) when creating clusters or using [Ambari](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-manage-ambari) on existing clusters. Apply the following configurations to your HDInsight cluster.
+
+!!! attention "Using Hive and the Feature Store"
+
+    HDInsight clusters cannot use their local Hive when being configured for the Feature Store as the Feature Store relies on custom Hive binaries and its own Metastore which will overwrite the local one. If you rely on Hive for feature engineering then it is advised to write your data to an external data storage such as ADLS from your main HDInsight cluster and use a second HDInsight cluster to read that data and access the Feature Store.
+
+Hadoop hadoop-env.sh:
+```
+export HADOOP_CLASSPATH=$HADOOP_CLASSPATH:/usr/lib/hopsworks/client/*
+```
+
+Hadoop core-site.xml:
+```
+hops.ipc.server.ssl.enabled=true
+fs.hopsfs.impl=io.hops.hopsfs.client.HopsFileSystem
+client.rpc.ssl.enabled.protocol=TLSv1.2
+hops.ssl.keystore.name=/usr/lib/hopsworks/keyStore.jks
+hops.rpc.socket.factory.class.default=io.hops.hadoop.shaded.org.apache.hadoop.net.HopsSSLSocketFactory
+hops.ssl.keystores.passwd.name=/usr/lib/hopsworks/material_passwd
+hops.ssl.hostname.verifier=ALLOW_ALL
+hops.ssl.trustore.name=/usr/lib/hopsworks/trustStore.jks
+```
+
+Spark spark-defaults.conf:
+```
+spark.executor.extraClassPath=/usr/lib/hopsworks/client/*
+spark.driver.extraClassPath=/usr/lib/hopsworks/client/*
+spark.sql.hive.metastore.jars=/usr/lib/hopsworks/apache-hive-bin/lib/*
+```
+
+Spark hive-site.xml:
+```
+hive.metastore.uris=thrift://MY_HOPSWORKS_INSTANCE_PRIVATE_IP:9083
+```
+
+!!! info
+    Replace MY_HOPSWORKS_INSTANCE_PRIVATE_IP with the private IP address of you Hopsworks Feature Store.
```

Comment:
We should be able to see the private_ip in hopsworks.ai in the details section for the cluster, but currently it's not there. A new issue for it?

Reply:
True. I'll add a ticket